ibm/mas_devops/roles/sls/tasks/install/main.yml

```diff
@@ -107,7 +108,16 @@
       password: "{{ mongocfg[0].stringData.password }}"
       hosts: "{{ mongocfg[1].spec.config.hosts }}"
       certificates: "{{ lookup('template', 'templates/mongo-certificates.yml.j2') }}"
-  when: mongocfg is defined
+  when: mongocfg is defined and mongocfg[0] is defined and mongocfg[0].stringData is defined
```

Comment:
what if we add an assert step also to validate if the mongo data needed for sls is set prior creating sls namespace or overriding the mongodb set facts?

Reply:
Code modified as per suggestion to add assert.